modules/auxiliary/admin/kerberos/keytab.rb

```diff
@@ -0,0 +1,201 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Kerberos keytab utilities',
+        'Description' => %q{
+          Utilities for interacting with keytab files, which can store the hashed passwords of one or
+          more principals.
+
+          Discovered keytab files can be used to generate Kerberos Ticket Granting Tickets, or bruteforced
+          offline.
+
+          Keytab files can be also useful for decrypting Kerberos traffic using Wireshark dissectors,
+          including the krbtgt encrypted blobs if the AES password hash is used.
+        },
+        'Author' => [
+          'alanfoster' # Metasploit Module
+        ],
+        'References' => [
+        ],
+        'License' => MSF_LICENSE,
+        'Notes' => {
+          'Stability' => [],
+          'SideEffects' => [],
+          'Reliability' => [],
+          'AKA' => []
```

Comment:
```suggestion
```
I don't think we enforce the 'AKA' key, should probably just be removed entirely if it's not going to have a value